fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2600,8 +2602,10 @@ void setupSimulatedSystem(std::vector<Future<Void>>* systemActors,
 			simHTTPMachines = deterministicRandom()->randomInt(1, 4);
 			fmt::print("sim http machines = {0}\n", simHTTPMachines);
 		}
+		int extraStorageMachineCount = testConfig.extraStorageMachineCountDC;
 
-		int totalMachines = machines + storageCacheMachines + blobWorkerMachines + simHTTPMachines;
+		int totalMachines =
+		    machines + storageCacheMachines + blobWorkerMachines + simHTTPMachines + extraStorageMachineCount;
```

Comment:
why not multiplying `datacenters` for `extraStorageMachineCount`?

Reply:
This is calculating total number of machines for each DC.